mzLib/Test/FileReadingTests/TestMsDataFile.cs

```diff
@@ -78,6 +80,26 @@ public static void TearDown()
             Console.WriteLine($"Analysis time: {Stopwatch.Elapsed.Hours}h {Stopwatch.Elapsed.Minutes}m {Stopwatch.Elapsed.Seconds}s");
         }
 
+        [Test]
+        public void TestOneBasedScanNumbersAndRetentionTimesAreInOrder()
```

Comment:
i don't understand this test.
Where are you using exampleRetentionTimeToSetUpArrays?

Reply:
test was not necessary with the last commit so I removed it